### DISCUSSION TITLE
Support of folder notes?

### DISCUSSION THREAD

**Discussion by andrey-jef** [oleeskild/obsidian-digital-garden#188] [Q&A]
Hi. Does the plugin currently support to publish folder notes (that is note named 'index.md', located inside a folder)?

It means when publishing, instead of displaying the 'index.md', it will display the equivalent folder's note

**Comment by uroybd** [comment]
It doesn't. However, taking `index.md` by default will be a mess. For example, I use `index.md` to create `db-folder` by the `db-folder` plugin.

I think, a better option can be add support to hoisting by permalink.

Let's say, I have a folder in the path `Personal/Musings`.

Now, I'll put a note in that dir like this:

```markdown
---
title: "All my musings"
dg-permalink: "/personal/musings/"
dg-publish: true
---

probably a dataview table here
```

Now, when the file tree generator sees there is a note for the permalink of that folder it will generate a structure like this:

- Personal
  - Musings
    - All my musings
    - Another note in default order
    - Then another
    - and so on...

@oleeskild what do you think?

**Reply by andrey-jef** [reply]
Absolutely, folder notes currently is not standardized. At the moment, there are [3 methods to implement and display folder notes](https://github.com/aidenlx/alx-folder-note/wiki/folder-note-pref), supported by [plugin](https://github.com/aidenlx/alx-folder-note) in Obsidian. I'm using the "index file, inside folder" method.

So if you suggest a "dg-permalink" approach, it will be a 4th option, and applicable only to dg-publishing feature.
